MIGRATION.md

```diff
@@ -0,0 +1,88 @@
+# Migrating to the `cloudant-node-sdk` library
+This document is to assist in migrating from the `nodejs-cloudant` (package: `@cloudant/cloudant`) to the newly supported [`cloudant-node-sdk`](https://github.com/IBM/cloudant-node-sdk) (package: `@ibm-cloud/cloudant`) that is compatible with Javascript and TypeScript.
+
+## Initializing the client connection
+There are several ways to create a client connection in `cloudant-node-sdk`:
+1. [Environment variables](https://github.com/IBM/cloudant-node-sdk#authentication-with-environment-variables)
+2. [External configuration file](https://github.com/IBM/cloudant-node-sdk#authentication-with-external-configuration)
+3. [Programmatically](https://github.com/IBM/cloudant-node-sdk#programmatic-authentication)
+
+[See the README](https://github.com/IBM/cloudant-node-sdk#code-examples) for code examples on using environment variables.
+
+## Other differences
+1. The way of storing credentials in a file has changed, the usage of `dotenv` package is not suggested. See the [external file configuration section](https://github.com/IBM/cloudant-node-sdk#authentication-with-external-configuration) in our API docs about the new handling.
+1. Fetching the database object first before performing operations on it is not required. For example, in the case of updating a document, this would mean calling first `getDocument` to fetch and after `putDocument` to update.
```

Comment:
`before performing operations on it` sounds a bit awkward.  What about:
```suggestion
1. Fetching the database object first before performing additional operations is not required. For example, in the case of updating a document you would first call `getDocument` to fetch and then `putDocument` to update.
```

Reply:
Okay. Changed 583c25f